src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -28,6 +29,13 @@ import zxcvbn from 'zxcvbn';
 import './SignUp.css';
 
 export default class SignUp extends Component {
+  componentDidMount() {
+    const cookies = new Cookies();
+    const isLoggedIn = !!cookies.get('loggedIn');
```

Comment:
Why **!!** ?

Reply:
Also, avoid the extra variable. use it directly in the if statement

Reply:
Okay done.